hello-world-master/source/components/scene/animationhelper.brs

```diff
@@ -0,0 +1,35 @@
+function translationAnimation(target as Object, originX as Integer, originY as Integer, newX as Integer, newY as Integer) as Object
+    translateAnim = createObject("roSGNode", "Animation")
+    translateAnim.duration = 1
+    translateAnim.easeFunction = "linear"
+    
+    vector2DFieldInterpolator = createObject("roSGNode", "Vector2DFieldInterpolator")
+    vector2DFieldInterpolator.key = [0.0, 1.0]
```

Comment:
try to use 3 keys of interpolation....it is more clear that it have a start point - a middle point to pass through - and an ending point